### PR TITLE
HP Admin repo UI Nixification

### PR DESCRIPTION
**Pull request by zo-el**
The UI is nixified, i.e. it is able to run seamlessly on a nix env `holo-nix` 
- [x] Script to Builds and run hp-admin
- [x] Script to Build and run holofuel
- [x] Script to Unit tests

    `<agent_id>::<happ_id>-<dna_alias>`

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -27,16 +42,16 @@ let
   multiInstanceConfig = dna: map (agent: {
     agent = agent.id;
     dna = dna.name;
-    id = dna.name+"::"+agent.id;
+    id = agent.public_address+"::<happ_id>-"+dna.name;
```
Comment (JettTech) [inline default.nix:45]: Love the standardization here. :100: :   Is there anyway we clarify happ_id further though (ie:` hha_id` for holo environments)?

File: default.nix [hunk 2]
```diff
@@ -27,16 +42,16 @@ let
...
-      path = ".holochain/holo/storage/${dna.name+"::"+agent.id}";
+      path = ".holochain/holo/storage/${agent.public_address+"::<happ_id>-"+dna.name}";
       type = "file";
     };
   }) config.agents;
 
 
   multiInterfaceInstanceConfig = dna: map (agent: {
-    id = dna.name+"::"+agent.id;
+    id = agent.public_address+"::<happ_id>-"+dna.name;
```
Comment (JettTech) [inline default.nix:54]: Same.

File: package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
     "start": "rm -rf ./node_modules/.cache/babel-loader && node scripts/start.js",
     "start:holofuel": "REACT_APP_HOLOFUEL_APP=true npm start",
     "build": "node scripts/build.js",
+    "build:holofuel": "REACT_APP_HOLOFUEL_APP=true node scripts/build.js",
```
Comment (JettTech) [inline package.json:9]: Yes. :rocket:

**Review by JettTech — APPROVED** [review]
Nice work. These were definitely needed additions. :+1:

File: default.nix (hunk 1 above)
Comment (zo-el) [reply default.nix:45]: Yes.

File: agent.key [hunk 4]
```diff
@@ -0,0 +1 @@
...xNTMsMjUxLDg1LDk3LDk5LDcsNDMsMjUyLDQ0LDExNCwyMTIsNTIsMTU2LDIzMywzNSw1LDE1MCw5NywxNzIsODRdLCJjaXBoZXIiOlsyMTEsNTcsMjYsNjgsNjEsNDcsNzUsOCw0MywxMTAsMjMzLDM3LDE1MCwxMzEsMTk3LDkwLDEyOSw0NSwxNzcsMjI5LDU4LDE3OCw3OCwxNzQsMTQ5LDIzMiwxOTksMTMwLDExNCwxNzksMTU2LDI0LDI4LDExMCwyMTIsMTM3LDI0NSwxNjQsNjksMTg3LDMsMTA2LDQ5LDE5MywyMDQsMTE2LDExLDEzMywxNSwxNjEsMjA5LDI3LDIzMiw2LDI4LDE3NCw4MSw0MiwxNjgsNzQsMTIzLDE5NSw5LDMxLDIsMTcxLDIwNSwyMDEsMSwxMjIsMTIyLDk0LDIyNiw1MywxNzIsMTU0LDIyMSwyMjMsNywxODUsMTExLDEzNCwxMDcsMjEwLDM0LDIyNSwxOTYsMTIsMTUxLDQxLDE1MiwxMzcsMjMzLDIyOCwxNDgsMTYwLDk0LDM4LDE3MywxNSwxMzAsMTMzLDMwLDE1OSwyMDMsMjAxLDUxLDUyLDk0LDcsMTUyLDYsMjAzLDE1LDkyLDE3Myw0NiwyNTMsMTc0LDc4XX0="},"root_seed":{"blob_type":"Seed","seed_type":"OneShot","hint":"","data":"eyJzYWx0IjpbMTg3LDE2NiwyNCw4Nyw0NSwxNDcsNTYsMjksNjAsMTA1LDgyLDEwMiwyNDcsMTMsMjExLDVdLCJub25jZSI6WzIyOSwyNDAsMTYzLDI0MiwxMzksMjA1LDUyLDg3LDEyNSwyNyw0MSwxMzYsOCwyMDksNjQsMTk0LDk2LDE3NiwxMjYsMTgzLDE3OCwyNTMsMjEyLDQwXSwiY2lwaGVyIjpbMTA3LDY0LDEyNyw3Myw4NSwyMzksNTQsMTA3LDEyOCwyMTEsMTM0LDgsMTc2LDczLDgwLDc1LDEyNSwxMzYsMTI0LDEwOCw3NCw3NCwxNDUsMTc5LDI1LDEzMyw0NCw3MCwxNjQsMCwxODgsMTU5LDE2NCwxOTgsMiw3MCw0NSw5OSwxODYsMjA1LDEwNSwxMjgsMzgsODMsNDksMTg1LDIwMiw1MF19"}}}
```
Comment (robbiecarlton) [inline agent.key:1]: what's the rationale for checking in the key here?

File: config.nix [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+
+  # Set Agents which is used to generate the conductor-config.toml
+  # Add any number of agents you want to spin up on the same conductor
+
+  agents =[{
```
Comment (robbiecarlton) [inline config.nix:6]: or equivalently, what's the rationale for checking in the config file?

**Review by robbiecarlton — APPROVED** [review]
Just a clarifying question, but otherwise LGTM

File: agent.key (hunk 4 above)
Comment (zo-el) [reply agent.key:1]: @robbiecarlton, This was done to smoothen running the tests on CI. 
if we don't have a default `config.nix` the CI would have to generate it. 

Using a default `config.nix` would easy development. 

This change doesn't change the fact that you can add more agents to the config to be used in the conductor.

**Pull request merged**